pom.xml

```diff
@@ -236,4 +241,4 @@
       </plugin>
     </plugins>
   </build>
-</project>
+</project>
```

Comment:
Can you configure your IDE to not remove the new line at the end of the file?

Reply:
Done